builtin/logical/pki/path_fetch.go

```diff
@@ -99,6 +109,11 @@ func pathFetchCRL(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: `crl(/pem|/delta(/pem)?)?`,
 
+		DisplayAttrs: &framework.DisplayAttributes{
+			OperationPrefix: operationPrefixPKI,
+			OperationSuffix: "crl|crl-pem|crl-delta|crl-delta-pem",
```

Comment:
I've applied the changes for crl, cert-raw and unified-crl in https://github.com/hashicorp/vault/pull/19422/commits/a0e8e3f31b8b71bc894e6a89ca2fc417d25ac229

However, I'm not sure if it's applicable to the issuer equivalents as those already have der/pem explicit suffixes:

| Path | OperationIDs | 
| ---- | ------------ | 
| "/{pki_mount_path}/issuer/{issuer_ref}/crl" | pki-issuer-read-crl |
| "/{pki_mount_path}/issuer/{issuer_ref}/crl/delta" | pki-issuer-read-crl-delta |
| "/{pki_mount_path}/issuer/{issuer_ref}/crl/delta/der" | pki-issuer-read-crl-delta-der |
| "/{pki_mount_path}/issuer/{issuer_ref}/crl/delta/pem" | pki-issuer-read-crl-delta-pem |
| "/{pki_mount_path}/issuer/{issuer_ref}/crl/der" | pki-issuer-read-crl-der |
| "/{pki_mount_path}/issuer/{issuer_ref}/crl/pem" | pki-issuer-read-crl-pem |

I can make the first one `pki-issuer-read-crl-der2` (since operation ID's must be unique) or just leave it as is. What do you think?